From b863b732e267081d5dbeb783472401a7bb40a44c Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Mon, 14 Jun 2021 10:42:16 +0100
Subject: [PATCH] Do not use the CMAKE_FIND_ROOT_PATH variable when looking for
 Tensorflow Source

CMAKE_FIND_ROOT_PATH variable is used in Yocto for cross-compiling.
CMake uses it to point to the root directory of the target environment
In Armnn. it requires the location of the tensorflow source code, therefore,
we need to override this behaviour when looking for tensorflow source code.

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
---
 cmake/GlobalConfig.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/GlobalConfig.cmake b/cmake/GlobalConfig.cmake
index 8a7c9a1..7c40946 100644
--- a/cmake/GlobalConfig.cmake
+++ b/cmake/GlobalConfig.cmake
@@ -233,7 +233,8 @@ endif()
 if(BUILD_TF_LITE_PARSER)
     find_path(TF_LITE_SCHEMA_INCLUDE_PATH
               schema_generated.h
-              HINTS ${TF_LITE_GENERATED_PATH})
+              HINTS ${TF_LITE_GENERATED_PATH}
+              NO_CMAKE_FIND_ROOT_PATH)
 
     message(STATUS "Tf Lite generated header found at: ${TF_LITE_SCHEMA_INCLUDE_PATH}")
 
-- 
2.7.4

